app/src/preview.js

```diff
@@ -119,7 +119,8 @@ class Preview extends React.Component {
           <Grid columns={12} gap='0' height='100%' alignContent='center'>
             <Cell width={10} left={2} className='box'>
               {this.state.isMatching ? (
-                <LoadingOverlay>Identifying locations&hellip;</LoadingOverlay>
+                <LoadingOverlay>Cleaning your dataset. Don't leave!<br />
+                This will take around 5-10 minutes.</LoadingOverlay>
```

Comment:
this kind of spacing would be more readable:
```
<LoadingOverlay>
  Cleaning your dataset. Don't leave!<br />
  This will take around 5-10 minutes.
</LoadingOverlay>
```